fastlane/Fastfile

```diff
@@ -15,21 +15,36 @@
 
 require 'yaml'
 
+files_with_version_number = [
+  './pubspec.yaml',
+  './ios/purchases_flutter.podspec',
+  './macos/purchases_flutter.podspec',
+  './ios/Classes/PurchasesFlutterPlugin.m',
+  './android/build.gradle',
+  './android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java'
+]
+repo_name = 'purchases-flutter'
+changelog_latest_path = './CHANGELOG-LATEST.md'
+changelog_path = './CHANGELOG.md'
+
 before_all do
   setup_circle_ci
   update_fastlane  
 end
 
-desc "Increment build number and update changelog"
-lane :bump_and_update_changelog do |options|
-  new_version_number = options[:version]
-  previous_version = current_version()
-
-  sh "fastlane ios bump version:#{new_version_number}"
-  sh "fastlane android bump version:#{new_version_number} current_version:#{previous_version}"
-  attach_changelog_to_master
-
-  increment_version_in_pubspec(previous_version, new_version_number)
+desc "Bump version, update swift header, edit changelog, and create pull request"
+lane :bump do |options|
+  bump_version_update_changelog_create_pr(
+    current_version: current_version_number,
+    changelog_latest_path: changelog_latest_path,
+    changelog_path: changelog_path,
+    files_to_update: files_with_version_number,
+    files_to_update_without_prerelease_modifiers: [],
```

Comment:
I checked and there are no `.plist` files that need updates on a new version here so this can be empty.